ludwig/backend/ray.py

```diff
@@ -163,6 +166,22 @@ def shutdown(self):
         self.executor.shutdown()
 
 
+class BatchInferModel:
+    def __init__(self, remote_model, predictor_kwargs, output_columns, features, data_hdf5_fp, *args, **kwargs):
+        self.model = remote_model.load()
```

Comment:
We need to define this class in-line because we don't want to load the remote model until we're running in the worker context.